FFXIVClientStructs/FFXIV/Client/UI/RaptureAtkModule.cs

```diff
@@ -42,6 +42,12 @@ public unsafe partial struct RaptureAtkModule {
     [MemberFunction("E8 ?? ?? ?? ?? 48 8B 46 58 48 85 C0")]
     public partial bool DecRefStringArrayData(int index);
 
+    [MemberFunction("E8 ?? ?? ?? ?? 66 89 46 50")]
+    public partial nint OpenAddon(uint addonNameId, uint valueCount, AtkValue* values, AgentInterface* parentAgent, long unk, uint ownerAddonId, int unk2);
+
+    [MemberFunction("E8 ?? ?? ?? ?? 0F B7 C0 48 83 C4 60")]
+    public partial nint OpenAddonByAgent(byte* addonName, AtkUnitBase* addon, int valueCount, AtkValue* values, AgentInterface* agent, nint a7, ushort parentAddonId);
```

Comment:
Return type is `void` not `nint`

Reply:
As far as I know, they addon id is always `ushort`.

Its also being passed like this (as word):
![Q6wZRCtVu4](https://github.com/aers/FFXIVClientStructs/assets/96642047/2ca56df0-01d9-4653-902e-9bfd9b0e55b2)
Which means the `AddonId` field in the [AgentInterface](https://github.com/aers/FFXIVClientStructs/blob/main/FFXIVClientStructs/FFXIV/Client/UI/Agent/AgentInterface.cs#L12) struct has the wrong type.

Reply:
Yeah, as I said in the main comment it was the first pass and I did it in maybe 5 to 10 minutes, so it was very rough.